wurst/file/PreloadIO.wurst

```diff
@@ -17,7 +17,7 @@ import ErrorHandling
 import StringUtils
 
 /** Maximum amount of packets per single readable file. */
-public constant PACKETS_PER_FILE = 16 	
+public constant PACKETS_PER_FILE = bj_MAX_PLAYER_SLOTS
```

Comment:
Can you explain how this is related to player slots?

Reply:
The current implementation relies on `SetPlayerName` to write/read packets to/from files. The previous limit was 16 'packets', equal to the number of players. Now that 1.29 supports 28 players, having it depend on the actual number of players as defined by the game better conveys the intent and is more reliable.